lib/reek/core/examiner.rb

```diff
@@ -1,7 +1,7 @@
-require_relative '../source/source_repository'
-require_relative 'warning_collector'
-require_relative 'smell_repository'
 require_relative 'tree_walker'
+require_relative 'smell_repository'
+require_relative 'warning_collector'
+require_relative '../source/source_repository'
```

Comment:
Hmmmm. We should at least have commented that. This change will soon be forgotten and the next refactoring PR might break it again.:)
@mvz how about updating this with a comment?